test/internet/test-dns-ipv4.js

```diff
@@ -39,119 +39,124 @@ function checkWrap(req) {
 }
 
 TEST(function test_resolve4(done) {
-  var req = dns.resolve4('www.google.com', function(err, ips) {
+  const req = dns.resolve4('www.google.com',
+  common.mustCall(function(err, ips) {
     if (err) throw err;
```

Comment:
Would you mind replacing all of these lines in the file with `assert.ifError(err);`